### PR TITLE
Upgrade our blog: 1. logo 2. favicon 3. part of blogs layout

### PR DESCRIPTION
**Pull request by 824zzy**
Please check out if any questions.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -68,3 +68,6 @@ tmp/
 package-lock.json
 *.bak
 t.*
+
+# Vscoded setting
+.vscode
```
Comment (huan) [inline .gitignore]: `.vscode/setting.json` should not be ignored...

File: .vscode/tags [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+help	../scripts/fit_image.py	/^                        help="the location of image folder or image")$/;"	kind:variable	line:94
+help	../scripts/fit_image.py	/^                        help="the ratio to resize image (recommend 0.7)")$/;"	kind:variable	line:100
+help	../scripts/fit_image.py	/^                        help="the target image hegiht(recommend 1080)")$/;"	kind:variable	line:98
+help	../scripts/fit_image.py	/^                        help="the target image width (recommend 1920)")$/;"	kind:variable	line:96
+is_img	../scripts/fit_image.py	/^        is_img = args.images.endswith('.jpg') or args.images.endswith($/;"	kind:variable	line:111
+os	../scripts/fit_image.py	/^import os$/;"	kind:namespace	line:19
+parser	../scripts/fit_image.py	/^    parser = argparse.ArgumentParser()$/;"	kind:variable	line:92
+re	../scripts/fit_image.py	/^import re$/;"	kind:namespace	line:22
+resize_image	../scripts/fit_image.py	/^def resize_image(image_path, args):$/;"	kind:function	line:75
+sys	../scripts/fit_image.py	/^import sys$/;"	kind:namespace	line:23
```
Comment (huan) [inline .vscode/tags]: Do we need this file to be uploaded?

I think it's a mistake, please correct me if I'm wrong.

File: docs/_includes/head/custom.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicon/apple-touch-icon.png">
```
Comment (huan) [inline docs/_includes/head/custom.html]: Does this file to add the favicon feature?

Can we split this PR to different PR to let us review the codes easier?

File: docs/_includes/scripts.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"
+>
+</script>
+<script
+  type="text/javascript"
+  charset="utf-8"
+  src="https://vincenttam.github.io/javascripts/MathJaxLocal.js"
+>
+</script>
+{% endif %}
```
Comment (huan) [inline docs/_includes/scripts.html]: This file is related to the Mathjax enhancement.

Can we separate this file to another PR for clear review as well?

File: .gitignore (hunk 1 above)
Comment (824zzy) [reply .gitignore]: My vscode always change `"python.pythonPath"` automatically.

Anyway, I will cancel that ignore.

File: .vscode/tags (hunk 2 above)
Comment (824zzy) [reply .vscode/tags]: This file is a byproduct of changing tags of blogs. It is kinda log.

I will delete it.

File: docs/_includes/head/custom.html (hunk 3 above)
Comment (824zzy) [reply docs/_includes/head/custom.html]: Next time I will notice that rule.

File: docs/_includes/scripts.html (hunk 4 above)
Comment (824zzy) [reply docs/_includes/scripts.html]: Next time I will notice that rule.

File: .vscode/tags [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+argparse	../scripts/fit_image.py	/^import argparse$/;"	kind:namespace	line:21
+args	../scripts/fit_image.py	/^    args = parser.parse_args()$/;"	kind:variable	line:101
+chose_proper_resolution	../scripts/fit_image.py	/^def chose_proper_resolution(args, old_width, old_height):$/;"	kind:function	line:26
+fit_image.py	../scripts/fit_image.py	1;"	kind:file	line:1
+help	../scripts/fit_image.py	/^                        help="the location of image folder or image")$/;"	kind:variable	line:94
+help	../scripts/fit_image.py	/^                        help="the ratio to resize image (recommend 0.7)")$/;"	kind:variable	line:100
+help	../scripts/fit_image.py	/^                        help="the target image hegiht(recommend 1080)")$/;"	kind:variable	line:98
+help	../scripts/fit_image.py	/^                        help="the target image width (recommend 1920)")$/;"	kind:variable	line:96
+is_img	../scripts/fit_image.py	/^        is_img = args.images.endswith('.jpg') or args.images.endswith($/;"	kind:variable	line:111
+os	../scripts/fit_image.py	/^import os$/;"	kind:namespace	line:19
```
Comment (huan) [inline .vscode/tags]: Should we delete this file in this PR?

File: docs/_includes/head/custom.html [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
+<link rel="manifest" href="/assets/images/favicon/site.webmanifest">
```
Comment (huan) [inline docs/_includes/head/custom.html]: Why do we need a manifest config for the favicon?

Could we just put a multi-sized favicon.ico file in the webroot to solve this problem?

File: .vscode/tags (hunk 5 above)
Comment (824zzy) [reply .vscode/tags]: Done～

File: docs/_includes/head/custom.html (hunk 6 above)
Comment (824zzy) [reply docs/_includes/head/custom.html]: I am afraid not. I have not found a proper place to put the favicon link. However, I find this solution [in this issue](https://github.com/mmistakes/minimal-mistakes/issues/949) that custom header via `_includes`.

Comment (huan) [reply docs/_includes/head/custom.html]: OK, let's try to use this solution.

P.S. It would be great if we can use a single favicon.ico file and just put it to the web root directory.

File: docs/_includes/scripts.html [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+<script
+  type="text/javascript"
+  charset="utf-8"
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"
+>
+</script>
+<script
+  type="text/javascript"
+  charset="utf-8"
+  src="https://vincenttam.github.io/javascripts/MathJaxLocal.js"
```
Comment (huan) [inline docs/_includes/scripts.html]: I can not see this file had been used anywhere.

Does this mean that this is a theme system file? if so, could you please make sure that you did not overwrite other functions?

Comment (huan) [reply docs/_includes/scripts.html]: I mean, the `docs/_includes/scripts.html`
